# An elevation raster map

Header header
string elevation_id
Raster raster